tests/integration/test_features_service.py

```diff
@@ -339,6 +341,26 @@ def test_feature_with_callback(self):
         self.assertEqual(resp.content_type, 'text/javascript')
         resp.mustcontain('cb_({')
 
+    def test_feature_geojson_geom_supported_srs(self):
+        bodId = 'ch.bafu.bundesinventare-bln'
+        featureId = self.getRandomFeatureId(bodId)
+
+        for srs in SUPPORTED_OUTPUT_SRS:
+            resp = self.testapp.get('/rest/services/ech/MapServer/%s/%s' % (bodId, featureId), params={'sr': srs, 'geometryFormat': 'geojson', 'returnGeometry': 'true'}, status=200)
+            self.assertEqual(resp.content_type, 'application/json')
+            self.assertEqual(resp.json['feature']['id'], featureId)
+            self.assertGeojsonFeature(resp.json['feature'], srs)
+
+    def test_feature_esirjson_geom_supported_srs(self):
+        bodId = 'ch.bafu.bundesinventare-bln'
+        featureId = self.getRandomFeatureId(bodId)
+
+        for srs in SUPPORTED_OUTPUT_SRS:
+            resp = self.testapp.get('/rest/services/ech/MapServer/%s/%s' % (bodId, featureId), params={'sr': srs, 'geometryFormat': 'esrijson', 'returnGeometry': 'true'}, status=200)
+            self.assertEqual(resp.content_type, 'application/json')
+            self.assertEqual(resp.json['feature']['id'], featureId)
+            self.assertEsrijsonFeature(resp.json['feature'], srs)
+
```

Comment:
maybe add a test for no srs specified (to check correct default value)?

Reply:
OK, added two tests (for EsriJson and GeoJson)